rasa/cli/test.py

```diff
@@ -83,20 +83,43 @@ def test_core(args: argparse.Namespace) -> None:
         )
 
     else:
-        test_compare(args.model, stories, output)
+        test_compare_core(args.model, stories, output)
 
 
 def test_nlu(args: argparse.Namespace) -> None:
     from rasa.test import test_nlu, test_nlu_with_cross_validation
+    import os
 
     nlu_data = get_validated_path(args.nlu, "nlu", DEFAULT_DATA_PATH)
     nlu_data = data.get_nlu_directory(nlu_data)
 
-    if not args.cross_validation:
+    if isinstance(args.config, list):
+        if len(args.config) == 1:
+            args.config = args.config[0]
+            config_path = os.path.abspath(args.config)
+
+            if os.path.isdir(config_path):
```

Comment:
Should we validate if the files inside the directory are actually config files? E.g. using `rasa.data.is_config_file`?

Reply:
`rasa.data.is_config_file()` just checks for the standard `config.yml` naming. 

That's incompatible with this at the moment - my comparison mode depends on users labelling their pipelines with descriptive names (e.g. `pretrained_embeddings_spacy.yml`).

Reply:
Ok. But then maybe something like:
- check if file is a valid yaml file
- check if file contains at least the keywords: pipeline and language

Reply:
Something similar to https://github.com/RasaHQ/rasa/blob/master/rasa/train.py#L425